cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs

```diff
@@ -902,12 +902,19 @@ runQueryStakeAddressInfoCmd
 
         requireNotByronEraInByronMode eraInMode
 
-        result <- lift (queryStakeAddresses eInMode sbe stakeAddr networkId)
+        (stakeRewardAccountBalances, stakePools) <- lift (queryStakeAddresses eInMode sbe stakeAddr networkId)
           & onLeft (left . QueryCmdUnsupportedNtcVersion)
           & onLeft (left . QueryCmdLocalStateQueryError . EraMismatchError)
 
-        pure $ do
-          writeStakeAddressInfo mOutFile $ DelegationsAndRewards result
+        stakeDelegDeposits <- lift (queryStakeDelegDeposits eInMode sbe stakeAddr)
+          & onLeft (left . QueryCmdUnsupportedNtcVersion)
+          & onLeft (left . QueryCmdLocalStateQueryError . EraMismatchError)
+
+        return $ do
+          writeStakeAddressInfo
```

Comment:
Is it worth updating the function name to say `writeStakeAddressInfoAndBalances`?